src/pages/Swap/Components/ChartTabs.tsx

```diff
@@ -2,27 +2,26 @@ import { useTranslation } from 'react-i18next'
 import { Flex } from 'rebass'
 import styled from 'styled-components'
 
+import { MouseoverTooltip } from '../../../components/Tooltip'
 import { useIsDesktop } from '../../../hooks/useIsDesktopByMedia'
 import { useRouter } from '../../../hooks/useRouter'
 import { ChartOptions } from '../../../state/user/reducer'
 
 export const ChartTabs = ({
   activeChartTab,
   setActiveChartTab,
-  hasBothCurrenciesInput,
 }: {
   activeChartTab: ChartOptions
   setActiveChartTab: (tab: ChartOptions) => void
-  hasBothCurrenciesInput: boolean
 }) => {
   const { navigate } = useRouter()
   const { t } = useTranslation('swap')
   const isDesktop = useIsDesktop()
-  const proOptionsDisabled = !hasBothCurrenciesInput || !isDesktop
 
   return (
-    <Root>
-      <Tab
+    <MouseoverTooltip placement="top" disabled={isDesktop} content="Disabled on mobile">
```

Comment:
@0xVenky can you validate the string please?

Reply:
Do you think we will have place for `Available only on desktop` It sounds a bit more positive than Disabled :(

Reply:
changed it you can check it out